fs/expose/dokan/__init__.py

```diff
@@ -37,7 +43,7 @@
 to subprocess.Popen::
 
     >>> from subprocess import PIPE
-    >>> mp = dokan.MountProcess(fs,"Q",stderr=PIPE)
+    >>> mp = dokan.MountProcess(fs, "Q", stderr=PIPE)
```

Comment:
...and this be using the "full drive path" i.e. `Q:\\` as in the earlier examples?